Classes/System/Logging/DebugWriter.php

```diff
@@ -50,6 +50,11 @@ public function write($level, $message, $data = [])
             return;
         }
 
+        $isPageIndexingRequest = $this->getIsPageIndexingRequest();
+        if ($isPageIndexingRequest) {
+            return;
```

Comment:
I think thats a bit hard to figure out here and would couple the logic of the logger to much to the indexer.
IMO it would make more sense to log to a different location in this case which can be done by registering a PSR LogWriter, right? So the information will still be logged to a compatible LogWriter but not output by this "custom" debug log writer. I would suggest to remove this custom debug log writer stuff anyways as it is not really state of the art.

Reply:
Yes,
there is already a issue #2777 for replacing old-school logger with PSR-3.

So, what you think, should we merge this, or you could provide a patch without losing the infos?